packages/api/src/controllers/stripe.ts

```diff
@@ -67,6 +78,16 @@ export const reportUsage = async (req, adminToken) => {
 };
 
 async function reportUsageForUser(req, user, adminToken) {
+  if (user.email.includes("@livepeer.org")) {
```

Comment:
This is exploitable by creating an email on any other domain that starts with livepeer.org e.g. `dontchargeme@livepeer.org.justcast.it`

Perhaps considering checking for a suffix instead. There's probably a function for it but I don't remember.

Additionally, you might want to ignore any user with `isTestUser` or `isAdmin` (or sth like that).

Reply:
Right! Should have been endsWith, changing it! nice catch